lib/feeder_importer.rb

```diff
@@ -0,0 +1,72 @@
+# import script for a podcast live in feeder & dovetail
+require 'prx_access'
+
+class FeederImporter
```

Comment:
Yeah, I'm a bit confused why when one would use these standalone libs to run an import against an existing API ... vs just using the CMS API (which can handle this on its own via the PodcastImport model).